app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PostsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def index; end
```

Comment:
This works, but for readability and consistency it would be better to use:

```suggestion
  def index
  end
```